src/main/java/com/blackducksoftware/integration/hub/rest/CredentialsRestConnection.java

```diff
@@ -105,7 +105,7 @@ public void clientAuthenticate()
                         response = getClient().newCall(request).execute();
                         logResponseHeaders(response);
                         if (!response.isSuccessful()) {
```

Comment:
Though I approve of this change, it was nice to learn about the numbers and what they can do!